spec/requests/posts/index_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :system do
+  describe 'GET /users/:id/posts' do
+    it 'should show the posts index page' do
+      get '/users/11/posts'
+      expect(response).to be_successful
+    end
+
+    it 'should render correct template' do
+      get '/users/11/posts'
+      expect(response.body).to include('index page for posts')
+    end
```

Comment:
- [ ] hi, In this test suite we are checking for the correct template, I think we have a matcher in rspec which is `render_template` which you can use to check if the path is rendering the correct template like :point_down: 
`expect(response.body).to render_template(:index)`

notice: once you specify `type: :request` the render template will be available